layers/core_checks/cc_ray_tracing.cpp

```diff
@@ -2097,3 +2097,361 @@ bool CoreChecks::ValidateDeferredOperation(VkDevice device, VkDeferredOperationK
     }
     return skip;
 }
+
+bool CoreChecks::PreCallValidateCmdBuildPartitionedAccelerationStructuresNV(
+    VkCommandBuffer commandBuffer, const VkBuildPartitionedAccelerationStructureInfoNV *pBuildInfo,
+    const ErrorObject &error_obj) const {
+    bool skip = false;
+    // validate partitionedAccelerationStructure feature must be enabled
+    if (!enabled_features.partitionedAccelerationStructure) {
+        skip |= LogError("VUID-vkCmdBuildPartitionedAccelerationStructuresNV-partitionedAccelerationStructure-10536", device,
```

Comment:
```suggestion
        skip |= LogError("VUID-vkCmdBuildPartitionedAccelerationStructuresNV-partitionedAccelerationStructure-10536", commandBuffer,
```

all of these need to at least print the VkCommandBuffer as an object